engine/src/main/java/com/sandboni/core/engine/finder/bcel/visitors/annotations/CucumberAnnotationProcessor.java

```diff
@@ -22,9 +22,9 @@
     @Override
     public boolean process(JavaClass jc, Context context) {
         TestVertex.Builder runnerBuilder = new TestVertex.Builder(jc.getClassName(), RUN_WITH, context.getCurrentLocation());
-        AnnotationEntry cucumberOptionsAnnotation = AnnotationUtils.getAnnotation(jc.getConstantPool(), jc::getAnnotationEntries, Annotations.TEST.CUCUMBER_OPTIONS.getDesc());
-        if (Objects.nonNull(cucumberOptionsAnnotation)) {
-            String features = getAnnotationParameter(cucumberOptionsAnnotation, FEATURES);
+        Optional<AnnotationEntry> cucumberOptionsAnnotation = AnnotationUtils.getAnnotation(jc.getConstantPool(), jc::getAnnotationEntries, Annotations.TEST.CUCUMBER_OPTIONS.getDesc());
+        if (cucumberOptionsAnnotation.isPresent()) {
```

Comment:
as an improvement: 
instead of:
        if (cucumberOptionsAnnotation.isPresent()) {....}
it is better:
        cucumberOptionsAnnotation.ifPresent( value --> doSmth(value))

Reply:
Noted for future improvement!